eventservice/api/controllers/events.py

```diff
@@ -347,11 +338,17 @@ def put(event_id):
 
 
 def patch(event_id):
-    auth_middleware.authorize(auth_middleware.ROKWIRE_EVENT_WRITE_GROUPS)
-
     if not ObjectId.is_valid(event_id):
         abort(400)
     req_data = request.get_json(force=True)
+
+    # check group event
+    if req_data.get('createdByGroupId'):
```

Comment:
Here also, I think we should use the event data from the database and not the incoming request to decide whether this is a group event or not. 

Since PATCH is a partial request, the request body may not contain the createdByGroupId field even though it's a group event. As you would know, it's guaranteed to contain only the required fields (title, startDate, and category).